src/store/reducers/commentReducers.js

```diff
@@ -0,0 +1,74 @@
+import {
+  FETCH_AN_ARTICLE,
+  FETCH_ARTICLE_COMMENTS,
+  FETCH_AN_ARTICLE_404,
+  CREATE_A_COMMENT,
+  CREATE_A_REPLY,
+  CREATE_COMMENT_SUCCESS,
+  CREATE_REPLY_SUCCESS,
+} from '../actions/actionTypes';
+
+const initialState = {
+  isFetchingArticle: true,
+  isFetchingComments: true,
+  commentCreated: false,
+  replyCreated: false,
+  creatingComment: true,
+  creatingReply: true,
+  createCommentError: false,
+  slug: '',
+  errors: null,
+  author: {},
+  comments: [],
+  newComment: {},
+};
+
+const commentsReducers = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_AN_ARTICLE:
+      return {
+        ...state,
+        slug: action.payload.slug,
+        article: action.payload,
+        isFetchingArticle: false,
```

Comment:
I think that when you are fetching an article, `isFetchingArticle` should be true.